src/AppleNews/Component/ConditionalComponent.php

```diff
@@ -0,0 +1,90 @@
+<?php
+declare(strict_types=1);
+
+namespace Triniti\AppleNews\Component;
+
+use Assert\Assertion;
+use Triniti\AppleNews\Condition;
+
+/**
+ * @link https://developer.apple.com/documentation/apple_news/conditionalcomponent
+ */
+class ConditionalComponent extends Component
```

Comment:
extending Component might not work here.  I see properties on Component that shouldn't be included on ConditionalComponent

Reply:
Gotcha, I've updated it to extend AppleNewsObject instead